app/src/main/java/com/vero/tasky/agenda/presentation/eventdetails/EventDetailsViewModel.kt

```diff
@@ -58,6 +61,7 @@ class EventDetailsViewModel @Inject constructor(
                 photos = emptyList(),
                 host = user.userId
             ),
+            isConnected = connectionHandler.isConnected(),
```

Comment:
What if the user losses conection while they are editing the item?
They'd still be able to add/remove photos